test/Faker/Provider/ProviderOverrideTest.php

```diff
@@ -112,6 +112,7 @@ public function testPhoneNumber($locale = null)
         $faker = Faker\Factory::create($locale);
 
         $this->assertRegExp(static::TEST_STRING_REGEX, $faker->phoneNumber);
+        $this->assertRegExp(static::TEST_STRING_REGEX, $faker->e164PhoneNumber);
```

Comment:
Also, testing that it's a string is too loose. Please validate that the returned value complies with the expected format.